Src/Newtonsoft.Json-for-Unity.Tests/Assets/SystemWebPolyfill/HttpUtility.cs

```diff
@@ -0,0 +1,670 @@
+            }
+            return result.ToArray ();
+        }
+
+        /// <summary>
+        /// Decodes an HTML-encoded string and returns the decoded string.
+        /// </summary>
+        /// <param name="s">The HTML string to decode. </param>
+        /// <returns>The decoded text.</returns>
+        public static string HtmlDecode (string s) 
+        {
+            if (s == null)
+                return null;
+            
+            using (var sw = new StringWriter ()) {
+                HttpEncoder.Current.HtmlDecode (s, sw);
+                return sw.ToString ();
+            }
+        }
+    
+        /// <summary>
+        /// Decodes an HTML-encoded string and sends the resulting output to a TextWriter output stream.
+        /// </summary>
+        /// <param name="s">The HTML string to decode</param>
+        /// <param name="output">The TextWriter output stream containing the decoded string. </param>
+        public static void HtmlDecode(string s, TextWriter output) 
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+                
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlDecode (s, output);
+            }
+        }
+
+        public static string HtmlEncode (string s)
+        {
+            if (s == null)
+                return null;
+            
+            using (var sw = new StringWriter ()) {
+                HttpEncoder.Current.HtmlEncode (s, sw);
+                return sw.ToString ();
+            }
+        }
+        
+        /// <summary>
+        /// HTML-encodes a string and sends the resulting output to a TextWriter output stream.
+        /// </summary>
+        /// <param name="s">The string to encode. </param>
+        /// <param name="output">The TextWriter output stream containing the encoded string. </param>
+        public static void HtmlEncode(string s, TextWriter output) 
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+                
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlEncode (s, output);
+            }
+        }
+        public static string HtmlEncode (object value)
+        {
+            if (value == null)
+                return null;
+
+#if !(MOBILE || NO_SYSTEM_WEB_DEPENDENCY || true)
+            IHtmlString htmlString = value as IHtmlString;
+            if (htmlString != null)
+                return htmlString.ToHtmlString ();
+#endif
+
+            return HtmlEncode (value.ToString ());
+        }
+
+        public static string JavaScriptStringEncode (string value)
+        {
+            return JavaScriptStringEncode (value, false);
+        }
+
+        public static string JavaScriptStringEncode (string value, bool addDoubleQuotes)
+        {
+            if (String.IsNullOrEmpty (value))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4755183)